### PR TITLE
[OTA] Update README with ACL examples

### LINKED ISSUE #15316: Add ACLs support for OTA
#### Problem
Once https://github.com/project-chip/connectedhomeip/pull/14921 merges, access control will be enforced. OTA does not have proper ACLs in place so any operations would result in a failure.

 #### Proposed Solution
Put proper ACLs in place for OTA

### PR DESCRIPTION
**Pull request by carol-apple**
#### Problem
There are no instructions to write ACL attribute for OTA purpose

Fixes: https://github.com/project-chip/connectedhomeip/issues/15316

#### Change overview
Add an example for test purpose

#### Testing
No functional change, readme update only

### CODE REVIEW

**Review by shubhamdp — APPROVED** [review]

File: examples/ota-provider-app/linux/README.md [hunk 1]
```diff
@@ -52,6 +52,26 @@ An example of the `--otaImageList` file contents:
...
 ```
 
+## Access Control Requirements
+
+Commissioner or Administrator SHOULD install necessary ACL entries at
+commissioning time or later to enable processing of QueryImage commands from OTA
+Requestors on their fabric, otherwise that OTA Provider will not be usable by
+OTA Requestors. For testing purpose, below is an example of how to write the ACL
+attribute with just one entry: Administer privilege, CASE authentication mode,
+wildcard subject (all nodes), wildcard target (all clusters on every endpoint).
```
Comment (bzbarsky-apple) [inline examples/ota-provider-app/linux/README.md:62]: Why would one ever do this for OTA Provider?  Wouldn't it at the very least set the cluster to "OTA Provider" even if leaving everything else as wildcards?  That would actually be a sane-ish ACL to use and then would not need the "real world scenarios" warning.

Comment (carol-apple) [reply examples/ota-provider-app/linux/README.md:62]: I was following @mlepage-google advice and leaving this as open as possible for testing. I could make it stricter to only the OTA Provider cluster as target.

Comment (tcarmelveilleux) [reply examples/ota-provider-app/linux/README.md:62]: Agree with @bzbarsky-apple: For the OTA provider, privilege only needs to be *Operate*, and only cluster OTA Provider. This is more realistic and safer.

**Review by andy31415 — APPROVED** [review]

**Review by harsha-rajendran — APPROVED** [review]

File: examples/ota-provider-app/linux/README.md (hunk 1 above)
Comment (carol-apple) [reply examples/ota-provider-app/linux/README.md]: Updated the example

**Pull request merged**